galaxy.yml

```diff
@@ -17,6 +17,7 @@ tags:
   - infrastructure
 dependencies:
   azure.azcollection: '*'
+  community.crypto: '2.9.0'
```

Comment:
It seems this is only a requirement for the tests, for the openssh keypair. Is there a way to use the test machinery to install it rather than make it a collection dependency?

Reply:
Update: https://github.com/redhat-cop/cloud.aws_ops/blob/main/tests/integration/requirements.yml provides an example of how to manage test-only dependencies

Reply:
Done